### PR TITLE
[IMP] hr_skills : Onboarding Screen

### PR DESCRIPTION
**Pull request by mepe-odoo**
When you go on odoo.com/trial you don't have demo data.

With this commit, when you install the Appraisal app without demo data, you can select a scenario to easily understand how to create an appraisal.

task: 3772927

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_attendance/data/scenarios/hr_attendance_scenario.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <record id="hr.employee_eg" model="hr.employee">
+            <field name="resource_calendar_id" ref="resource_calendar_std_38h"/>
+            <field name="barcode">456</field>
+        </record>
+
+        <record id="hr.employee_sj" model="hr.employee">
+            <field name="pin">789</field>
+        </record>
+    </data>
+</odoo>
```
Comment (mepe-odoo) [inline addons/hr_attendance/data/scenarios/hr_attendance_scenario.xml:41]: forget end line

**Comment by tivisse** [comment]
@robodoo r+ rebase-ff

**Comment by tivisse** [comment]
@robodoo override=ci/security

Rationale: The usage to convert_file has been moved actually. But there is this task to remove the calls completely: https://www.odoo.com/odoo/1251/tasks/4047727?cids=1

**Comment by tivisse** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**